test/system/workarea/storefront/order_history_system_test.rb

```diff
@@ -18,7 +18,7 @@ def test_viewing_legacy_orders_in_account_summary
 
         click_link t('workarea.storefront.orders.view')
         assert_current_path(storefront.users_order_path('LO12345'))
-        assert(page.has_content?(order.total_price))
+        assert(page.has_content?("$53.69"))
```

Comment:
This will fail if an implementer switches the default currency, is there another way? Can we use the `number_to_currency` helper maybe since that's what the view would be using to output?

Reply:
This is how we do it in all other tests like this. I can change it, but we're just going to have this problem everywhere else if they change default currency.

Reply:
Ah ok, well that was a bad decision :(